apps/hash-graph/lib/authorization/src/zanzibar.rs

```diff
@@ -5,234 +5,223 @@ use std::borrow::Cow;
 
 use serde::{Deserialize, Serialize};
 
-/// The relation or permission of a [`Subject`] to an [`Resource`].
+/// The relation or permission of a [`Resource`] to another [`Resource`].
 pub trait Affiliation<R: Resource + ?Sized>: AsRef<str> {}
 
-/// A computed set of [`Subject`]s for a particular [`Resource`].
+impl<A: Affiliation<R>, R: Resource> Affiliation<R> for &A {}
+// impl<A: Affiliation<R>, R: Resource> Affiliation<&R> for A {}
```

Comment:
Why is this commented out, and if it is not needed why is it still here?

Reply:
I forgot to remove that, thank you

Reply:
Removed in da0abcf0a3792aff1edcd21cd699fa2dbb623886. I expect to remove all reference-based parameters as all types should implement `Clone` anyway and this avoids issues like passing reference to tuples and such.